src/pages/my_page/MyPagePage.tsx

```diff
@@ -96,16 +86,14 @@ const MyPagePage = () => {
   const typeQuery =
     selectedTypesArray.length > 0 ? `(${selectedTypesArray.map((type) => 'type:' + type).join(' OR ')})` : '';
 
-  const selectedStatusesArray = Object.entries(selectedStatuses)
-    .filter(([_, selected]) => selected)
-    .map(([key]) => key);
+  const ticketStatusValues: TicketStatus[] = ['New', 'Pending', 'Closed', 'Completed'];
+  const selectedStatusesArray = searchParams.get(TicketSearchParam.Status)?.split(',') || ticketStatusValues;
```

Comment:
denne får bare typen `string[]` pga query param value nå. Kan være greit å legge til en `as TicketStatus[]` bare for å ha det på plass. tror det blir riktig iaf 🤔 
```suggestion
  const selectedStatusesArray = (searchParams.get(TicketSearchParam.Status)?.split(',') ?? ticketStatusValues) as TicketStatus[];
```

Reply:
fixed